packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -149,6 +149,7 @@ function convertGraphSeriesToDataFrame(graphSeries: GraphSeriesXY): DataFrame {
 }
 
 function convertJSONDocumentDataToDataFrame(timeSeries: TimeSeries): DataFrame {
+  const values: TimeSeriesValue[][] = [];
   const fields = [
```

Comment:
sorry to nitpick, but I think this would be better to just

```ts
const fields: Field[] = [
  {
    ...
    values: []
```

that way we don't need to declare the extra variable up top just to type the property.

Reply:
yeah that's nicer 👍